code/datums/diseases/advance/symptoms/heal.dm

```diff
@@ -143,7 +143,10 @@
 	if(!active_coma)
 		return
 	active_coma = FALSE
-	M.cure_fakedeath("regenerative_coma")
+	if(deathgasp)
+		M.cure_fakedeath("regenerative_coma")
+	else
+		M.SetUnconscious(0)
```

Comment:
Why does it only remove the fakedeath/coma traits if it was under the deathgasp threshold?

Reply:
So before, it would do the whole fake death scheme regardless of the threshold. This makes it so the fake death is only applicable with the threshold, and as such it should only ever need to remove it if the threshold is active.

Reply:
Fake death is the threshold that decides whether they _death gasp_ or not. Not whether they fall into a regen Coma

Reply:
> 99% sure Fake death is the threshold that decides whether they _death gasp_ or not. Not whether they fall into a regen Coma

I don't think I was clear. The way the coma was started always did fake death regardless of the deathgasp, at least as it was told to me. This should fix it.

Probably an issue with it using fake_death("") to cause the coma.